### DISCUSSION TITLE
telebot.types: refactor for Dictionaryable class.

### DISCUSSION THREAD

**Discussion by vb64** [eternnoir/pyTelegramBotAPI#1977] [Ideas]
At this manner: https://github.com/vb64/telemulator3/blob/main/telemulator3/dictionaryable.py

This will reduce the amount of code in telebot.types and make it easier to add Dictionaryable functionality to the other telebot.types classes.

If it is deemed useful, I can prepare a PR.

**Comment by coder2020official** [comment]
I had the same idea but I don't think it is good to think of refactoring types file now, however it is a good idea but may lead to some breaking changes. @Badiboy Ur opinion?

**Comment by Badiboy** [comment]
I don't think it will bring something valuable. It may reduce some code in types, but that code is not a significant problem...

**Comment by vb64** [comment]
In my opinion, useful, popular and long-lived projects like this one (by the way, many thanks for the great work), the ease of code maintenance becomes a key value.

And code amount reduction plays an important role here.
